modules/installation-azure-marketplace-subscribe.adoc

```diff
@@ -0,0 +1,136 @@
+:ipi:
+endif::[]
+ifeval::["{context}" == "installing-azure-user-infra"]
+:upi:
+endif::[]
+
+//mpytlak: The procedure differs depending on whether this module is used in an IPI or UPI assembly.
+
+:_content-type: PROCEDURE
+[id="installation-azure-marketplace-subscribe_{context}"]
+= Selecting an Azure Marketplace image
+If you are deploying an {product-title} cluster using the Azure Marketplace offering, you must first obtain the Azure Marketplace image. The installation program uses this image to deploy worker nodes. When obtaining your image, consider the following:
+
+* While the images are the same, the Azure Marketplace publisher is different depending on your region. If you are located in North America, specify `redhat` as the publisher. If you are located in EMEA, specify `redhat-limited` as the publisher.
+* The offer includes a `rh-ocp-worker` SKU and a `rh-ocp-worker-gen1` SKU. The `rh-ocp-worker` SKU represents a Hyper-V generation version 2 VM image. The default instance types used in {product-title} are version 2 compatible. If you are going to use an instance type that is only version 1 compatible, use the image associated with the `rh-ocp-worker-gen1` SKU. The `rh-ocp-worker-gen1` SKU represents a Hyper-V version 1 VM image.
+
+.Prerequisites
+
+* You have installed the Azure CLI client `(az)`.
+* Your Azure account is entitled for the offer and you have logged into this account with the Azure CLI client.
+
+.Procedure
+
+. Display all of the available {product-title} images by running one of the following commands:
+** North America:
++
+--
+[source,terminal]
+----
+$  az vm image list --all --offer rh-ocp-worker --publisher redhat -o table
+----
+.Example output
+[source,terminal]
+----
+Offer          Publisher       Sku                 Urn                                                             Version
+-------------  --------------  ------------------  --------------------------------------------------------------  --------------
+rh-ocp-worker  RedHat          rh-ocp-worker       RedHat:rh-ocp-worker:rh-ocpworker:4.8.2021122100               4.8.2021122100
+rh-ocp-worker  RedHat          rh-ocp-worker-gen1  RedHat:rh-ocp-worker:rh-ocp-worker-gen1:4.8.2021122100          4.8.2021122100
+----
+--
+** EMEA:
++
+--
+[source,terminal]
+----
+$  az vm image list --all --offer rh-ocp-worker --publisher redhat-limited -o table
+----
+.Example output
+[source,terminal]
+----
+Offer          Publisher       Sku                 Urn                                                             Version
+-------------  --------------  ------------------  --------------------------------------------------------------  --------------
+rh-ocp-worker  redhat-limited  rh-ocp-worker       redhat-limited:rh-ocp-worker:rh-ocp-worker:4.8.2021122100       4.8.2021122100
+rh-ocp-worker  redhat-limited  rh-ocp-worker-gen1  redhat-limited:rh-ocp-worker:rh-ocp-worker-gen1:4.8.2021122100  4.8.2021122100
```

Comment:
Sorry to add a late change here, because I know this is due soon. 

We should probably add a note/sentence somewhere to avoid potential confusion about versions (because the versioning here is confusing). 

The output of this command shown in this table will have the correct version to be used. In this case it is `4.8.2021122100`. What is potentially confusing to users is that the correct version here is going to be 4.8.x even though they could be installing 4.9, 4.10, or 4.11. We only distribute the 4.8 image, and it doesn't really matter because the VM image will be automatically updated to the latest image for their target version before the install completes.

So I think we probably need a note saying: even though you're installing 4.9(+) the correct image version to specify will be 4.8.x and your VMs will automatically be upgraded as part of the install.

Reply:
That is an excellent catch @patrickdillon. My latest commit adds a note to step 1. Please let me know what you think.